Packs/Armorblox/pack_metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Armorblox",
+    "description": "Armorblox is an API-based platform that stops targeted email attacks, protects sensitive data, and automates incident response.",
+    "support": "partner",
+    "currentVersion": "1.0.0",
+    "author": "Armorblox",
+    "url": "https://www.armorblox.com/",
+    "email": "support@armorblox.com",
+    "devEmail": "app-publishers@armorblox.com",
+    "categories": [
+        "Email Gateway"
+    ],
+    "tags": [
+        "Alerts", "Attack", "Breach", "Compliance", "Email", "Forensics", "GDPR","HIPAA","Incident Response","IT","Machine Learning","Malware","PII","Scam","Security","Security Analytics","Spam","Threat Intelligence"
+
+    ],
+    "useCases": ["Email Communication"],
+    "keywords": ["Email Security", "Cloud Security", "Abuse Mailbox", "Data Loss Protection", "Data Leaks", "Compliance"],
+    "githubUser": [
```

Comment:
What was this field for?

Reply:
(Optional) List of Github usernames to receive notification in the PR in case pack files were modified.
https://xsoar.pan.dev/docs/packs/packs-format#pack_metadatajson

Reply:
Please add `urajat`, `ab-samba` & `ab-dj` in the array for this field.